qa/setup_packages.py

```diff
@@ -352,11 +352,11 @@ def get_pyvers_name(self, url, cuda_version):
 
 all_packages = [PlainPackage("opencv-python", ["4.5.1.48"]),
                 CudaPackage("cupy",
-                        { "100" : ["8.0.0"],
-                          "110" : ["8.0.0"] },
+                        { "100" : ["8.6.0"],
```

Comment:
There is no 9.0.0 for CUDA 110 for Linux (I see only prebuild for windows). Downgrading to 8.6.0.